packages/web-app-dicom-viewer/src/App.vue

```diff
@@ -1028,4 +1039,11 @@ export default defineComponent({
 
   border-bottom: 1px solid var(--oc-color-border);
 }
+
+@media only screen and (max-width: 959px) {
```

Comment:
try to use oc variables instead of pixel values if possible. I think it's this one: var(--oc-breakpoint-medium-default)